### PR TITLE
Update information-schema-data-lock-waits.md

### PR DESCRIPTION
**Pull request by longfangsong**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- Other reference link(s):
   [English Version of this PR](https://github.com/pingcap/docs/pull/9533)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@longfangsong Is this a doc enhancement?

**Comment by longfangsong** [comment]
> @longfangsong Is this a doc enhancement?

It's document about an [enhancement](https://github.com/pingcap/tidb/pull/33993) I've done on TiDB recently and planed to be released with TiDB 6.2

**Comment by Oreoxmt** [comment]
@MyonKeminta PTAL

**Review by cfzjywxk — APPROVED** [review]

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by MyonKeminta — APPROVED** [review]

File: information-schema/information-schema-data-lock-waits.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ DESC data_lock_waits;
 > **警告：**
 >
 > * 仅拥有 [PROCESS](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_process) 权限的用户可以查询该表。
-> * 目前该表仅能记录**悲观锁等锁**的情况，如果乐观事务（如 autocommit 的事务）被悲观锁阻塞，这一表中不会显示相应等锁信息。 
+> * 由于实现限制，目前对于乐观事务被阻塞的情况 `SQL_DIGEST` 和 `SQL_DIGEST_TEXT` 字段为 `null`，如果需要知道相关的 SQL，可以将此表与 [`CLUSTER_TIDB_TRX`](/information-schema/information-schema-tidb-trx.md) 进行 `JOIN` 来获得对应事务的所有 SQL。
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply information-schema/information-schema-data-lock-waits.md:42]: +1

**Review by TomShawn — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**